Makefile

```diff
@@ -21,7 +21,7 @@ checks: ## Run the application checks
 checks: yamllint otherchecks
 
 pipenv.timestamp:
-	pipenv install
+	pipenv sync
```

Comment:
The `touch $@` is missing or the `.PHONY` and should be renamed ...

Reply:
Done and added dependency on Pipfile.lock.

Reply:
Note that in GMF 2.6 we install the whole Pipfile.lock just for yamllint.